docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -499,7 +506,7 @@ oc patch openstackdataplanenodeset openstack --type='json' --patch='[
 +
```

Comment:
Can customers name the validation service anything they want? And can the OpenStackDataPlaneDeployment  be called anything, or does it have to be openstack-pre-adoption?